pom.xml

```diff
@@ -109,11 +109,58 @@
     </dependency>
   </dependencies>
 
+  <profiles>
+    <profile>
+      <id>snapshot-deployment</id>
+      <activation>
+        <activeByDefault>false</activeByDefault>
+      </activation>
+      <distributionManagement>
+        <snapshotRepository>
+          <id>bds-artifactory</id>
+          <url>https://artifactory.internal.synopsys.com/artifactory/bds-integrations-snapshot</url>
```

Comment:
Should we expose synopsys internal artifactory URLs?

@dmamidisynopsys

Reply:
@devmehtasynopsys We never should. Use pipeline environment variables.

Reply:
Also, remove other gradle references. See details below under Jenkins Security Scan / security-scan / scan (pull_request)

Reply:
will do thanks